### PR TITLE
mt7621: Fix EX400 imagebuilder

### PR DESCRIPTION
**Pull request by msandber**
A factory image for this device depends on an initramfs image and they were explicitly removed from the imagebuilder recently. Now the factory image creation fails miserably and it also affects custom image creation with the firmware selector.

Add the initramfs kernel to the staging so that it's shipped with the
imagebuilder. Also remove a image build target added solely for DNA EX400.

Tested by creating a factory and syspupgrade images locally with
the imagebuilder and verified their functionality.

Related work
  c85348d9abf4 ("imagebuilder: remove initramfs image files")

Fixes: fea2264d9fdd ("ramips: mt7621: Add DNA Valokuitu Plus EX400")

### CODE REVIEW

File: target/linux/ramips/image/mt7621.mk [hunk 1]
```diff
@@ -2352,7 +2354,7 @@ define Device/openfi_5pro
   $(Device/dsa-migration)
   IMAGE_SIZE := 65216k
   DEVICE_VENDOR := OpenFi
-  DEVICE_MODEL := 5Pro 
+  DEVICE_MODEL := 5Pro
```
Comment (musashino205) [inline target/linux/ramips/image/mt7621.mk:2357]: Please separate this change to the another commit.

File: target/linux/ramips/image/mt7621.mk [hunk 2]
```diff
@@ -926,7 +926,7 @@ define Device/dlink_dir-2150-r1
   KERNEL := $$(KERNEL)
   IMAGES += factory.bin
   IMAGE/factory.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi | \
-	check-size | sign-dlink-ru e6587b35a6b34e07bedeca23e140322f 
+	check-size | sign-dlink-ru e6587b35a6b34e07bedeca23e140322f
```
Comment (musashino205) [inline target/linux/ramips/image/mt7621.mk:929]: ditto

**Comment by hauke** [comment]
Would a change like the one done in #16740 also work?

**Comment by musashino205** [comment]
How about using `append-image-stage` with `initramfs-kernel.bin`?

`kernel-initramfs-bin | lzma | uImage lzma` --> `append-image-stage initramfs-kernel.bin`

**Comment by msandber** [comment]
use append-image-stage as suggested by @musashino205, remove Build/kernel-initramfs-bin target as it was added for this device only.

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!
